Demo/ImagePickerDemo/ImagePickerDemo/Info.plist

```diff
@@ -24,6 +24,8 @@
 	<true/>
 	<key>UILaunchStoryboardName</key>
 	<string>LaunchScreen</string>
+	<key>UIMainStoryboardFile</key>
```

Comment:
Well the demo isn't really prone to change in this case because everything is isolated inside the `ImagePickerController`. It makes for a cuter demo when there are just a few lines to setup.
And third, not a big fan of Storyboards in iOS development because I don't use them myself... ever :sunglasses: 

This is my take on it :grin: